lib/Reporters/MutationTestingElementsReporter.cpp

```diff
@@ -76,9 +63,7 @@ json11::Json createFiles(const Result &result,
           sourceInfoProvider.getSourceInfo(mutationPoint);
 
       std::string status =
-          (killedMutants.find(mutationPoint) == killedMutants.end())
-              ? "Survived"
-              : "Killed";
+          (killedMutants.count(mutationPoint) == 0) ? "Survived" : "Killed";
```

Comment:
The primary reason for the change: given that the intention here is `check if contains` I believe that `count` is more semantically correct than the `find`.
The secondary reason for the change: I didn't measure, but I assume that comparison of numbers (`== 0`) is more efficient than the comparison of iterators since the iterators are classes, which may involve either function calls, or heavy template code inlining.